src/rgw/rgw_zone_types.h

```diff
@@ -545,12 +545,13 @@ WRITE_CLASS_ENCODER(RGWZoneGroupPlacementTierS3)
 enum GlacierRestoreTierType : uint8_t {
```

Comment:
please use enum class; and "none" should be the firset element in the enumeration, with value 0